.github/workflows/publish-stable.yml

```diff
@@ -114,6 +114,16 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+  comment:
+    name: Comment on PRs
+    runs-on: buildjet-4vcpu-ubuntu-2204
+    needs: [tag, npm]
+    if: needs.tag.outputs.is_stable == 'true'
+    steps:
+      - uses: nflaig/release-comment-on-pr@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@wemeetagain as noted in the [usage section](https://github.com/marketplace/actions/comment-on-pull-requests-included-in-release#usage) of the gh action, this token needs the `public_repo` permission which is not the case by default. Might need to create a [personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) in that case and use that one instead.

![image](https://user-images.githubusercontent.com/38436224/223111599-c9607e14-c4be-4976-932e-8c106b7d70ce.png)

Reply:
We have a secret named GH_PAGES_TOKEN which should have that permission

Reply:
wondering if we can dy-run that somehow, the comment will then be added by the github pages bot account I would assume, so the only requirement is that the account is able to post comments.